specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -133,7 +133,7 @@
             "in": "body",
             "name": "payload",
             "schema": {
-              "$ref": "#/definitions/Offer"
+              "$ref": "#/definitions/OfferProperties"
```

Comment:
Why are you using `OfferProperties` instead of `Offer` on the PUT? This doesn't appear to adhere to the ARM resource contract - we expect the PUT to contain a resource body (name/type/id, properties wrapped in "properties").

Reply:
@anthony-c-martin , changes were made, please review